### DISCUSSION TITLE
Payer and access to repos

### DISCUSSION THREAD

**Discussion by oaigudmundsson** [community/community#22153] [New to GitHub]
<p>Hello,</p>
<p>Lets say company A och company B work together. A is paying the github bill for B. Since B don’t like A access to the repos. Can B blocks A from accessing repos? or only to allow A access some repos…</p>
<p>Is this possible?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>You absolutely can separate billing access and repository access. To do this, you assign the person responsible for paying the bills as a “Billing manager”. They will have full access to the payment information and receipts but no repository access.</p>
<p><a href="https://help.github.com/github/setting-up-and-managing-organizations-and-teams/adding-a-billing-manager-to-your-organization" class="onebox" target="_blank">https://help.github.com/github/setting-up-and-managing-organizations-and-teams/adding-a-billing-manager-to-your-organization</a></p>

**Comment by oaigudmundsson** [comment]
<p>Hello!</p>
<p>Thanks for the info. I will try this solution tomorrow!</p>
<p>Cheers</p>
